extensions/wikia/Helios/User.class.php

```diff
@@ -252,6 +252,11 @@ public static function onUserLogout() {
 		return true; // So that wfRunHooks evaluates to true.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)